topics/osint/index.md

```diff
@@ -0,0 +1,11 @@
+---
+topic: osint
+related: malfrats
+aliases: investigation
+display_name: OSINT
+logo: osint.png
```

Comment:
What is "malfrats industries"?

Reply:
It is a collective of developers of well used open-source OSINT tools !

Reply:
Is there an image that might better represent OSINT? As someone new to the topic, it's not clear to me how that logo connects to OSINT.